yt_dlp/extractor/mxplayer.py

```diff
@@ -9,15 +10,15 @@ class MxplayerIE(InfoExtractor):
         'url': 'https://www.mxplayer.in/show/watch-my-girlfriend-is-an-alien-hindi-dubbed/season-1/episode-1-online-9d2013d31d5835bb8400e3b3c5e7bb72',
         'info_dict': {
             'id': '9d2013d31d5835bb8400e3b3c5e7bb72',
+            'display_id': 'episode-1-online',
             'ext': 'mp4',
             'title': 'Episode 1',
-            'description': 'md5:62ed43eb9fec5efde5cf3bd1040b7670',
+            'description': "Watch Season 1, Episode 1 of the show My Girlfriend Is an Alien (Hindi Dubbed).'",
```

Comment:
Why did you change some descriptions from md5 to readable text? 🤔

Reply:
This one is not too long. So it is fine

Reply:
Fetching the description from description tag in json. It typically has some form of description of the video. Currently I couldn't find md5 anywhere in the json being fetched.

Reply:
The md5 checksum is computed by the test runner, taking the field as input (in this case "description"). As pukkandan said, in this case is okay because it's short but if it were a description with >150 characters we would prefer md5

https://www.md5hashgenerator.com/
![image](https://user-images.githubusercontent.com/26639800/192404452-0d4d79dd-c4f8-4a93-859c-174982739fee.png)

Reply:
Ok. So should I change all descriptions to md5 hash? I believe there are some descriptions longer than 150.

Reply:
Ok. This is done. Have replaced all descriptions for the extractor with md5 hashes.